main.go

```diff
@@ -21,18 +21,23 @@ func main() {
 	r.HandleFunc("/expand", ExpandHandler).Methods("POST")
 	r.HandleFunc("/parser", ParserHandler).Methods("POST")
 
-	var host, certFile, keyFile string
-	var port int
-	flag.StringVar(&host, "listen-host", "0.0.0.0", "Listen host")
-	flag.IntVar(&port, "listen-port", 8080, "Listen port")
-	flag.StringVar(&certFile, "cert-file", "", "SSL Cert file")
-	flag.StringVar(&keyFile, "key-file", "", "SSL Key file")
-	listenSpec := fmt.Sprintf("%s:%d", host, port)
+	host := os.GetEnv("LISTEN_HOST")
+	if host == "" {
+		host = "0.0.0.0"
+	}
+	port := os.GetEnv("LISTEN_PORT")
+	if port == "" {
+		port = "5000"
```

Comment:
makes sense, can you submit one more commit with the port set to 8080.

maybe you guys can use https://github.com/direnv/direnv and set your port to 5000 then your code can stay the same. Just a thought. :)